### DISCUSSION TITLE
播放器弹幕与视频弹幕数量不匹配

### DISCUSSION THREAD

**Discussion by AvoidGuest** [ywmoyue/biliuwp-lite#306] [General]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/107521025/248dcdeb-7942-4f2c-a9ec-1b87c01a34ef)

**Comment by GD-Slime** [comment]
按照 https://github.com/ywmoyue/biliuwp-lite/issues/162#issuecomment-1605818362 的说法，这里显示的是按6分钟为一段的本段所显示的弹幕数量。

**Reply by AvoidGuest** [reply]
可在手机端能正常显示的弹幕在BiliLite上没有

**Reply by ywmoyue** [reply]
我知道问题在哪了，网页端登录没登录获取到的弹幕数量不一样，bililite上请求弹幕是没带登录信息的

**Reply by ywmoyue** [reply]
我处理下吧

**Comment by ywmoyue** [comment]
4.6.14 已修复
